components/Tabs/src/Theming/MDCTabs+MaterialTheming.m

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCTabs+MaterialTheming.h"
+
+#import <CoreGraphics/CoreGraphics.h>
+#import <UIKit/UIKit.h>
+
+#import <MaterialComponents/MaterialColorScheme.h>
+#import <MaterialComponents/MaterialTypographyScheme.h>
+
+static const CGFloat kUnselectedOpacity = (CGFloat)0.6;
+
+@implementation MDCTabBar (MaterialTheming)
+
+- (void)applyPrimaryThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme {
+  id<MDCColorScheming> colorScheme = scheme.colorScheme;
+  [self applySemanticThemeWithColorScheme:colorScheme];
+
+  id<MDCTypographyScheming> typographyScheme = scheme.typographyScheme;
+  [self applyThemeWithTypographyScheme:typographyScheme];
+}
+
+- (void)applySurfaceThemeWithScheme:(nonnull id<MDCContainerScheming>)scheme {
+  id<MDCColorScheming> colorScheme = scheme.colorScheme;
+  [self applySurfaceThemeWithColorScheme:colorScheme];
+
+  id<MDCTypographyScheming> typographyScheme = scheme.typographyScheme;
+  [self applyThemeWithTypographyScheme:typographyScheme];
+}
+
+- (void)applySemanticThemeWithColorScheme:(id<MDCColorScheming>)colorScheme {
```

Comment:
This should be "Primary" and not "Semantic".

Reply:
Addressed in 4e8cb3cc3ec6eb9a0b5536dac519b6b866791415